ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/forms/useReportFormValues.tsx

```diff
@@ -36,6 +41,11 @@ export type CVEStatus = 'FIXABLE' | 'NOT_FIXABLE';
 
 export type CVESDiscoveredSince = 'ALL_VULN' | 'SINCE_LAST_REPORT' | 'START_DATE';
 
+export type DeliveryDestination = {
+    notifier: NotifierIntegration | null;
```

Comment:
This caught my eye, and when I looked, I found this type is a union of all possible notifiers.

https://github.com/stackrox/stackrox/blob/b5e8e156671b384e411485ac21b7c8c16349fd0d/ui/apps/platform/src/types/notifier.proto.ts#L5-L14

Should we only import and specify `EmailNotifierIntegration` for now?